tests/e2e/scripts/run_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+eval openvpn --config /etc/openovpn/config.ovpn --auth-user-pass /etc/openovpn/authfile  &>/dev/null &
+sleep 10
+#ssh -i /config/$1  -o "StrictHostKeyChecking no" $2@$3 'cd k3s && git pull --rebase origin master'
+ssh -i /config/$1  -o "StrictHostKeyChecking no" $2@$3 '/usr/local/go/bin/go get github.com/onsi/ginkgo/v2'
```

Comment:
```suggestion
ssh -i /config/"$1"  -o 'StrictHostKeyChecking no' "$2@$3" '/usr/local/go/bin/go get github.com/onsi/ginkgo/v2'
```

Handle eventual whitespaces in variables and no double quotes for static strings.